proto/penumbra/penumbra/crypto/decaf377_frost/v1alpha1/decaf377_frost.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+
+package penumbra.crypto.decaf377_frost.v1alpha1;
```

Comment:
Having this as a standalone package seems fine. I think though that we should decide where the _Penumbra_ threshold messages should go. (By this I mean messages where the input to the signing ceremony is a `TransactionPlan` and the output is an `AuthorizationData`, rather than the "raw" FROST where the input is an arbitrary message and the output is a signature).

Previously we'd stubbed these out as part of Narsil, but I think we may want to put them in the `custody` package and put the threshold custody implementation as part of the `custody` crate. WDYT?

Reply:
I was going to put them in `custody.threshold`